tooling/car/fixture_test.go

```diff
@@ -93,9 +93,54 @@ func TestIssue54MustGetChildren(t *testing.T) {
 	//     }
 	//   ]
 	// }
-	cids = f.MustGetChildrenCids("sub1")
+	cids = f.MustGetDescendantsCids("sub1")
 	assert.Len(t, cids, 3)
 	assert.Equal(t, "QmZgfvZtoFdbJy4JmpPHc1NCXyA7Snim2L8e6zKspiUzhu", cids[0])
 	assert.Equal(t, "QmaATBg1yioWhYHhoA8XSUqD1Ya91KKCibWVD4USQXwaVZ", cids[1])
 	assert.Equal(t, "QmdQEnYhrhgFKPCq5eKc7xb1k7rKyb3fGMitUPKvFAscVK", cids[2])
 }
+
+func TestMustGetChildrenDespiteMissingBlocks(t *testing.T) {
+	f := MustOpenUnixfsCar("./_fixtures/file-3k-and-3-blocks-missing-block.car")
```

Comment:
:thought_balloon:  excellent call to have a separate fixture for testing tooling, independent of fixture used in actual gateway test :ok_hand: